tstest/integration/integration_test.go

```diff
@@ -528,22 +529,31 @@ func (n *testNode) AwaitListening(t testing.TB) {
 	}
 }
 
-func (n *testNode) AwaitIP(t testing.TB) (ips string) {
+func (n *testNode) AwaitIPSlice(t testing.TB) (ipslice []string) {
```

Comment:
no need to put "Slice" in the name of the method or even name the result parameter.

How about `AwaitIPs(t testing.TB) []netaddr.IP` ?